src/archivematicaCommon/lib/elasticsearch/aipfile_mets_mapping.json

```diff
@@ -0,0 +1,474 @@
+{
+  "properties": {
+    "dmdSec": {
+      "properties": {
+        "ns0:xmlData_dict_list": {
+          "properties": {
+            "@xmlns:ns1": {
+              "type": "string"
+            },
+            "@xmlns:ns0": {
+              "type": "string"
+            },
+            "@xmlns:xsi": {
+              "type": "string"
+            },
+            "ns1:dublincore_dict_list": {
+              "properties": {
+                "dc:provenance": {
+                  "type": "string"
+                },
+                "dc:date": {
+                  "type": "string"
+                },
+                "@xsi:schemaLocation": {
+                  "type": "string"
+                },
+                "dc:title": {
+                  "type": "string"
+                },
+                "dc:description": {
+                  "type": "string"
+                }
```

Comment:
That appears to be the case; these were both generated after processing the same AIP!